documentation/API-Testing-Guidelines.md

```diff
@@ -127,6 +127,24 @@ For the Feature description, API name and version must be included. When the fea
 Feature: CAMARA Device location verification API, v0.2.0 - Operation verifyLocation
 ```
 
+#### Feature Context
+
+Every feature will include a context after the `Feature` tag to provide relevant information about the implementation and execution of the tests.
+
+For the feature context, the following template should be used:
+
+```
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * 
+  #
+  # Testing assets:
+  # * 
+  #
+  # References to OAS spec schemas refer to schemas specifies in {apiname}.yaml, version {version}
```

Comment:
As the API version is already indicated at the top of the Feature file, I would not add it again here, as it creates additional maintenance work when creating releases (vX.Y.Z -> vwip -> v....)

Reply:
Yes, good point @jlurien, that is already stated within `Feature` tag so better to avoid duplication and additional maintenance work